### PR TITLE
Added a note for the fast export flag in export data  help msg

### PR DESCRIPTION
**Pull request by priyanshi-yb**
Added a note for Env var in the `export data` help msg.
Screenshot 2023-05-23 at 2 55 59 PM

### CODE REVIEW

File: yb-voyager/cmd/exportData.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ import (
 
 var exportDataCmd = &cobra.Command{
 	Use:   "data",
-	Short: "This command is used to export table's data from source database to *.sql files",
+	Short: "This command is used to export table's data from source database to *.sql files \nNote: For Oracle and MySQL, there is an experimental feature to speed up the data export, set the env var EXPERIMENTAL_FAST_EXPORT=1 for trying that out",
```
Comment (shubham-yb) [inline yb-voyager/cmd/exportData.go:41]: LGTM.
nit: Can we write `environment variable` in full instead of `env var`?

**Review by shubham-yb — APPROVED** [review]

File: yb-voyager/cmd/exportData.go [hunk 2]
```diff
@@ -38,7 +38,7 @@ import (
 
 var exportDataCmd = &cobra.Command{
 	Use:   "data",
-	Short: "This command is used to export table's data from source database to *.sql files",
+	Short: "This command is used to export table's data from source database to *.sql files \nNote: For Oracle and MySQL, there is an experimental feature to speed up the data export, set the environment variable EXPERIMENTAL_FAST_EXPORT=1 for trying that out",
```
Comment (makalaaneesh) [inline yb-voyager/cmd/exportData.go:41]: Let's also point the user to the documentation? For now, it's not live, so we can just point the user to the documentation. 

How about:

`Note: For Oracle and MySQL, there is an experimental feature to speed up the data export, set the environment variable EXPERIMENTAL_FAST_EXPORT=1 to try it out. You can refer to Voyager Documentation for more details - https://docs.yugabyte.com/preview/migrate/`

**Review by makalaaneesh — APPROVED** [review]
LGTM with small comment

File: yb-voyager/cmd/exportData.go (hunk 2 above)
Comment (priyanshi-yb) [reply yb-voyager/cmd/exportData.go:41]: or directly to the export-data section, there will be a note in the Export data section itself for this right?
https://docs.yugabyte.com/preview/migrate/migrate-steps/#export-data

Comment (makalaaneesh) [reply yb-voyager/cmd/exportData.go:41]: @priyanshi-yb yup, that's better!

**Pull request merged**